### ISSUE TITLE
Make sure all reports are still working

### ISSUE THREAD

**Issue by itsrachelfish** [playasoft/laravel-voldb#78]
There have been considerable schema changes in the past 3 months and we need to make sure that all of the reporting features are still working properly. I just tried to get a "shifts filled" report from production and was greeted with an error message.

![laravel-error](https://cloud.githubusercontent.com/assets/1670549/25321571/d0e69b0c-286c-11e7-8fae-54ee9eabd131.png)

## Manual tests

- [x] User report - all
- [x] User report - individual
- [x] Department report - all
- [x] Department report - indvidual
- [x] Misc report - hours by user
- [x] Misc report - shifts filled

**Comment by itsrachelfish** [comment]
Was looking through the error log, looks like this is the culprit:

```
[2017-04-24 03:33:00] prod.ERROR: exception 'ErrorException' with message 'Division by zero' in laravel-voldb/laravel/app/Http/Controllers/ReportController.php:466
```

**Comment by itsrachelfish** [comment]
This is fixed as of 5c1c05b7dd0867502497e2d29f6fbc8a6ff1393c
